jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -113,7 +113,8 @@
             (progn
               (ros::ros-info "[main] ~a, ~a" arm state)
               (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
-              (send *ri* :move-arm-body->bin arm target-bin :scale 5.0)
+              (send *baxter* :avoid-shelf-pose arm target-bin)
+              (send *ri* :send-av 5000)
```

Comment:
I think it is better to use `angle-vector-sequence` like https://github.com/start-jsk/jsk_apc/blob/master/jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l#L602-609